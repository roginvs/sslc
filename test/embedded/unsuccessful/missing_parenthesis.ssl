procedure start begin
  variable x;
  x = (1+;
end

