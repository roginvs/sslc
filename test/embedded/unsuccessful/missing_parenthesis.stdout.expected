Set include directory: ../include
Compiling missing_parenthesis.ssl
[Error] <missing_parenthesis.ssl>:3:11: Expression expected.

*** THERE WERE ERRORS (1 of them) ***
